### ISSUE TITLE
"Bad tester" - please do not blame people, blame code

### ISSUE THREAD

**Issue by tessi** [lemurheavy/coveralls-public#79]
I am using coveralls for OpenProject (https://coveralls.io/r/opf/openproject) and I am quite happy so far. However, I see a paragraph "Bad tester"/"Good tester" on the overview page which I do not find constructive nor polite.

I really don't like to blame people. Who is responsible for less code coverage? The commiter, the person who pushed the commit, the person who merged, a tool that failed? Furthermore, does it help to blame people to get a constructive discussion about better coverage? I think not. It embarrasses people (whether a coverage drop is their fault or not), which is not a good starting point for a constructive discussion.

I prefer to discuss code. Which pull request lead to coverage changes? Which part of the code is uncovered? Is there code that was covered first but then uncovered?

To sum it up: I wish there was a way to disable (or remove) the "Bad tester"/"Good tester" part.

Thanks,
Tessi
